mount.go

```diff
@@ -132,6 +144,35 @@ func mount(source, destination, fsType string, flags int, options string) error
 	return nil
 }
 
+// Allocate hugepages by writing to sysfs
+func allocateHugePages(options string) error {
+
+	agentLog.WithField("HugePagesOption", options).Info("HugePages option string")
+
+	//options eg "pagesize=2048,size=107374182"
+	opt := strings.Split(options, ",")
+	pageSizeStr := strings.TrimPrefix(opt[0], "pagesize=")
```

Comment:
I can see this aligns with https://github.com/kata-containers/runtime/pull/3109 in terms of the order of the options. But this would break if the runtime switched the option order.

To protect against this, I'd rather we loop over `opt` and look for the `pagesize=` and `size=` prefixes instead.

Reply:
Sure I'll change it. I was hoping to avoid looping since we control the options string passed by the runtime.